packages/esm-service-queues-app/src/queue-table/queue-table.test.tsx

```diff
@@ -202,13 +205,15 @@ describe('QueueTable', () => {
     });
 
     const rows = screen.queryAllByRole('row');
-    const firstRow = rows[1];
-    const cells = within(firstRow).getAllByRole('cell');
+    expect(rows).toHaveLength(5);
+
+    const briansRow = rows[1];
+    const alicesRow = rows[3];
```

Comment:
The rows with data have to get target using higher indices because of the hidden collapsed rows.